src/main/js/store/reducers/users.js

```diff
@@ -59,7 +59,7 @@ const users = (state = initialState, action) => {
             });
         case USER_MANAGEMENT_USER_FETCHING_ALL:
             return Object.assign({}, state, {
-                inProgress: true,
+                inProgress: false,
```

Comment:
Do we actually want this set to false here?

Reply:
Yes, we want to separate the inProgress and fetching booleans. I have separated our use of the two flags. InProgress now refers to a change being made to a User or Role (create, edit, delete) and fetching refers to retrieving them.